File: src/WESTDemo.Domain/Models/Learner.cs
using System.Collections.Generic;

namespace WESTDemo.Domain.Models
{
    public class Learner : Entity
    {
        public int UserId { get; set; }
        public int GroupId { get; set; }
        
        //Navigation Property
        public ICollection<LearnerStatus> LearnerStatus { get; set; }
        public Group Group { get; set; }
        public User User { get; set; }
    }
}

File: src/WESTDemo/src/app/_models/Learner.ts
import { Course } from "./Course";
import { Group } from "./Group";

export class Learner {
    id: number;
    userId: number;
    groupId: number;
    firstname: string;
    username: string;
    lastname: string;
    group: Group;
    status: boolean;
    learnerStatus: Course[];
}